### ISSUE TITLE
BSEC 1.4.9.2 support

### ISSUE THREAD

**Issue by pficheux** [alexh-name/bsec_bme680_linux#32]
The "BSEC_1.4.8.0_Generic_Release" version defined in "make.config" is no longer available.

Is there any support for the current version (1.4.9.2) available from https://www.bosch-sensortec.com/software-tools/software/bsec ?

Thx by advance

**Comment by alexh-name** [comment]
Yeah, Bosch officially only supplies the most recent version. Unfortunately the new versions are no longer C only, but C++ and I am (currently) unwilling to change things over to C++ in my free time. PRs welcome though.
If you are lucky you might find an untempered version of BSEC =< 1.4.8.0 from somewhere else (I’ve seen downloads floating around on their forums), because Bosch’s license forbids individuals to distribute them. I might put some hashes of the versions I still have in the Readme though so you can at least check if they have been tempered with or not.

**Comment by pficheux** [comment]
Thanks for the quick answer ! I don't like Bosch policy regarding software.  I bought another device from Kitronik based on the same sensor but using a Pi Pico. The software is written in MicroPython, open source, well documented :-)
https://kitronik.co.uk/blogs/resources/pico-smart-air-quality-board-using-the-bme688-sensor

**Comment by bsimmo** [comment]
But that doesn't use the BSEC, just normal sensor control, you can do that with a Pi and Python easily, e.g. with Pimoroni interpretation.
https://github.com/pimoroni/bme680-python

Don't these use BSEC2 now if you go the BSEC route. Not looked for a while.

**Comment by alexh-name** [comment]
I just added the checksums of prior versions.

**Comment by ilsicca** [comment]
Hi! you can upload the BSEC_1.4.8.0_Generic_Release.zip?

**Comment by alexhof-shyftplan** [comment]
> Hi! you can upload the BSEC_1.4.8.0_Generic_Release.zip?

Unfortunately no. Bosch’s license forbids individuals to distribute them.

**Comment by alexh-name** [comment]
> Hi! you can upload the BSEC_1.4.8.0_Generic_Release.zip?

Unfortunately no. Bosch’s license forbids individuals to distribute them.

**Comment by HankB** [comment]
I just downloaded the 1.4.8 version of the library from the Bosch web site:

```text
hbarta@olive:~$ ls -lrt ~/Downloads/bsec_1*
-rw-r--r-- 1 hbarta hbarta 5618846 Aug 22 13:52 /home/hbarta/Downloads/bsec_1-4-8-0_generic_release_updated_v3.zip
hbarta@olive:~$ 
```

Edit: Further I can confirm that the new library builds if the first line in `make.config` is tweaked to point to the right directory:

```text
BSEC_DIR='./src/BSEC_1.4.8.0_Generic_Release_updated_v3'
```

**Comment by hydrogen602** [comment]
Here's the download for people looking for 1.4.8.0: https://www.bosch-sensortec.com/software-tools/software/previous-bsec-software-versions/

Works as of Aug 24, 2023

**Comment by mfs85** [comment]
Hi, why can't I compile?
I have BSEC version 1.4.8.0 but it gives below error, any idea why?
I'm compiling this on RPI 4, 64bit os.

```
$ ./make.sh
Patching...
patching file bsec_integration.c
Compiling...
/usr/bin/ld: skipping incompatible ./src/BSEC_1.4.8.0_Generic_Release_updated_v3/algo/normal_version/bin/RaspberryPi/PiThree_ArmV8/libalgobsec.a when searching for -lalgobsec
/usr/bin/ld: cannot find -lalgobsec: No such file or directory
collect2: error: ld returned 1 exit status
```
